rakelib/test.rake

```diff
@@ -4,7 +4,8 @@ namespace "test" do
     LogStash::Environment.bundler_setup!({:without => []})
     require "rspec/core/runner"
     require "rspec"
-    RSpec::Core::Runner.run([*args])
+    exit_code = RSpec::Core::Runner.run([*args])
```

Comment:
because calling fail is the standard way in rspec, you can redefine what this means whenever you want.